feg/gateway/services/session_proxy/servicers/session_controller_test.go

```diff
@@ -600,7 +600,7 @@ func TestSessionControllerTimeouts(t *testing.T) {
 	).Return(nil).Run(func(args mock.Arguments) {
 		done := args.Get(1).(chan interface{})
 		request := args.Get(2).(*gy.CreditControlRequest)
-		if request.RequestNumber%1 == 0 {
+		if request.RequestNumber%2 == 1 {
```

Comment:
Why did this test need to be updated?

Reply:
The statement was trivially true before and the check made no sense. Any integer modulo 1 will be zero.

Reply:
Oh yes, good point :sweat_smile: